ngraph/core/include/openvino/op/color_convert_nv12.hpp

```diff
@@ -0,0 +1,88 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/op/op.hpp"
+#include "openvino/op/util/attr_types.hpp"
+
+namespace ov {
+namespace op {
+namespace v8 {
+/// \brief Color conversion operation from NV12 to RGB/BGR format.
+///    Input:
+///        - Operation expects input shape in NHWC layout.
+///        - Input NV12 image can be represented in a two ways:
+///            a) Single plane: NV12 height dimension is 1.5x bigger than image height. 'C' dimension shall be 1
+///            b) Two separate planes: Y and UV. In this case
+///               b1) Y plane has height same as image height. 'C' dimension equals to 1
+///               b2) UV plane has dimensions: 'H' = image_h / 2; 'W' = image_w / 2; 'C' = 2.
+///        - Supported element types: u8, f32.
+///    Output:
+///        - Output node will have NHWC layout and shape HxW same as image spatial dimensions.
```

Comment:
How to make preprocessing produce `NCHW` output?

Currently, preprocessing (at least in VPUX plugin) is able to handle `NHWC` and `NCHW` in the case of output.

Reply:
This is going to be a separate preprocessing operation, like `convert_layout(const Layout& dst)`
So if user needs to convert from NV12 to RGB and have NCHW output, they will need to write `PreProcessSteps().convert_color(RGB).convert_layout("NCHW")`